packages/mui-joy/src/Button/Button.tsx

```diff
@@ -1,12 +1,12 @@
-import * as React from 'react';
-import PropTypes from 'prop-types';
-import { unstable_capitalize as capitalize, unstable_useForkRef as useForkRef } from '@mui/utils';
 import { useButton } from '@mui/base/ButtonUnstyled';
-import { useSlotProps } from '@mui/base/utils';
 import composeClasses from '@mui/base/composeClasses';
+import { useSlotProps } from '@mui/base/utils';
+import { unstable_capitalize as capitalize, unstable_useForkRef as useForkRef } from '@mui/utils';
+import PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
@hbjORbj is this on purpose or done by some tool?

I think `react` should always be at the top and then `prop-types` (just to follow the convention)

Reply:
I think my prettier does that. Resolved!